<p>Error on Jan 28, 2016 16:14PM - require(/var/www/html/projet/app/views/connexion.php): failed to open stream: No such file or directory in /var/www/html/projet/app/Core/View.php on line 34</p><p>Error on Jan 28, 2016 16:14PM - require(/var/www/html/projet/app/views/connexion.php): failed to open stream: No such file or directory in /var/www/html/projet/app/Core/View.php on line 34</p><p>Error on Feb 03, 2016 13:32PM - Illegal string offset 'title' in /var/www/html/projet/app/templates/default/header.php on line 23</p><p>Error on Feb 03, 2016 13:32PM - Illegal string offset 'title' in /var/www/html/projet/app/templates/default/header.php on line 23</p><p>Error on Feb 03, 2016 13:32PM - Illegal string offset 'title' in /var/www/html/projet/app/templates/default/header.php on line 23</p><p>Error on Feb 03, 2016 13:32PM - Illegal string offset 'title' in /var/www/html/projet/app/templates/default/header.php on line 23</p><p>Error on Feb 03, 2016 13:32PM - Illegal string offset 'title' in /var/www/html/projet/app/templates/default/header.php on line 23</p><p>Error on Feb 03, 2016 13:33PM - Illegal string offset 'title' in /var/www/html/projet/app/templates/default/header.php on line 23</p><p>Error on Feb 03, 2016 22:29PM - call_user_func_array() expects parameter 1 to be a valid callback, class 'Controllers\Scores' does not have a method 'index' in F:\wamp\www\projet\app\Core\Router.php on line 126</p><p>Error on Feb 03, 2016 22:29PM - call_user_func_array() expects parameter 1 to be a valid callback, class 'Controllers\Scores' does not have a method 'index' in F:\wamp\www\projet\app\Core\Router.php on line 126</p><p>Error on Feb 03, 2016 22:29PM - call_user_func_array() expects parameter 1 to be a valid callback, class 'Controllers\Scores' does not have a method 'index' in F:\wamp\www\projet\app\Core\Router.php on line 126</p><p>Error on Feb 04, 2016 9:37AM - call_user_func_array() expects parameter 1 to be a valid callback, class 'Controllers\Connexion' does not have a method 'inscription' in F:\wamp\www\projet\app\Core\Router.php on line 126</p><h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 04, 2016 17:45PM</p>

           <p><strong>Message:</strong> SQLSTATE[42S02]: Base table or view not found: 1146 La table 'projet.personne' n'existe pas SELECT * FROM personne WHERE login=?</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> F:\wamp\www\projet\app\Helpers\DB\DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 F:\wamp\www\projet\app\Helpers\DB\Query.php(62): Helpers\DB\DBManager->execute(Array)
#1 F:\wamp\www\projet\app\Models\Queries\PersonneSQL.php(15): Helpers\DB\Query->execute()
#2 F:\wamp\www\projet\app\Controllers\Connexion.php(101): Models\Queries\PersonneSQL->prepareFindByLogin('TestUser')
#3 [internal function]: Controllers\Connexion->inscription()
#4 F:\wamp\www\projet\app\Core\Router.php(126): call_user_func_array(Array, Array)
#5 F:\wamp\www\projet\app\Core\Router.php(217): Core\Router::invokeObject('Controllers\\Con...')
#6 F:\wamp\www\projet\app\Core\routes.php(35): Core\Router::dispatch()
#7 F:\wamp\www\projet\index.php(81): require('F:\\wamp\\www\\pro...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 04, 2016 17:45PM</p>

           <p><strong>Message:</strong> SQLSTATE[42S02]: Base table or view not found: 1146 La table 'projet.personne' n'existe pas SELECT * FROM personne WHERE login=?</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> F:\wamp\www\projet\app\Helpers\DB\DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 F:\wamp\www\projet\app\Helpers\DB\Query.php(62): Helpers\DB\DBManager->execute(Array)
#1 F:\wamp\www\projet\app\Models\Queries\PersonneSQL.php(15): Helpers\DB\Query->execute()
#2 F:\wamp\www\projet\app\Controllers\Connexion.php(101): Models\Queries\PersonneSQL->prepareFindByLogin('TestUser')
#3 [internal function]: Controllers\Connexion->inscription()
#4 F:\wamp\www\projet\app\Core\Router.php(126): call_user_func_array(Array, Array)
#5 F:\wamp\www\projet\app\Core\Router.php(217): Core\Router::invokeObject('Controllers\\Con...')
#6 F:\wamp\www\projet\app\Core\routes.php(35): Core\Router::dispatch()
#7 F:\wamp\www\projet\index.php(81): require('F:\\wamp\\www\\pro...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 04, 2016 17:49PM</p>

           <p><strong>Message:</strong> SQLSTATE[42S22]: Column not found: 1054 Champ 'login' inconnu dans where clause SELECT * FROM personne WHERE login=?</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> F:\wamp\www\projet\app\Helpers\DB\DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 F:\wamp\www\projet\app\Helpers\DB\Query.php(62): Helpers\DB\DBManager->execute(Array)
#1 F:\wamp\www\projet\app\Models\Queries\PersonneSQL.php(15): Helpers\DB\Query->execute()
#2 F:\wamp\www\projet\app\Controllers\Connexion.php(101): Models\Queries\PersonneSQL->prepareFindByLogin('TestBD')
#3 [internal function]: Controllers\Connexion->inscription()
#4 F:\wamp\www\projet\app\Core\Router.php(126): call_user_func_array(Array, Array)
#5 F:\wamp\www\projet\app\Core\Router.php(217): Core\Router::invokeObject('Controllers\\Con...')
#6 F:\wamp\www\projet\app\Core\routes.php(35): Core\Router::dispatch()
#7 F:\wamp\www\projet\index.php(81): require('F:\\wamp\\www\\pro...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 04, 2016 17:53PM</p>

           <p><strong>Message:</strong> SQLSTATE[42S22]: Column not found: 1054 Champ 'email' inconnu dans where clause SELECT * FROM personne WHERE email=?</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> F:\wamp\www\projet\app\Helpers\DB\DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 F:\wamp\www\projet\app\Helpers\DB\Query.php(62): Helpers\DB\DBManager->execute(Array)
#1 F:\wamp\www\projet\app\Controllers\Connexion.php(109): Helpers\DB\Query->execute()
#2 [internal function]: Controllers\Connexion->inscription()
#3 F:\wamp\www\projet\app\Core\Router.php(126): call_user_func_array(Array, Array)
#4 F:\wamp\www\projet\app\Core\Router.php(217): Core\Router::invokeObject('Controllers\\Con...')
#5 F:\wamp\www\projet\app\Core\routes.php(35): Core\Router::dispatch()
#6 F:\wamp\www\projet\index.php(81): require('F:\\wamp\\www\\pro...')
#7 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 04, 2016 20:43PM</p>

           <p><strong>Message:</strong> SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO)</p>

           <p><strong>Code:</strong> 1045</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 22</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/DBManager.php(22): PDO->__construct('mysql:host=loca...', 'root', '', Array)
#1 /var/www/html/projet/app/Helpers/DB/DBManager.php(35): Helpers\DB\DBManager->__construct()
#2 /var/www/html/projet/app/Helpers/DB/Query.php(43): Helpers\DB\DBManager::getInstance()
#3 /var/www/html/projet/app/Controllers/Connexion.php(22): Helpers\DB\Query->__construct()
#4 /var/www/html/projet/app/Core/Router.php(124): Controllers\Connexion->__construct(NULL)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 04, 2016 20:43PM</p>

           <p><strong>Message:</strong> SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO)</p>

           <p><strong>Code:</strong> 1045</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 22</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/DBManager.php(22): PDO->__construct('mysql:host=loca...', 'root', '', Array)
#1 /var/www/html/projet/app/Helpers/DB/DBManager.php(35): Helpers\DB\DBManager->__construct()
#2 /var/www/html/projet/app/Helpers/DB/Query.php(43): Helpers\DB\DBManager::getInstance()
#3 /var/www/html/projet/app/Controllers/Connexion.php(22): Helpers\DB\Query->__construct()
#4 /var/www/html/projet/app/Core/Router.php(124): Controllers\Connexion->__construct(NULL)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 04, 2016 20:43PM</p>

           <p><strong>Message:</strong> SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO)</p>

           <p><strong>Code:</strong> 1045</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 22</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/DBManager.php(22): PDO->__construct('mysql:host=loca...', 'root', '', Array)
#1 /var/www/html/projet/app/Helpers/DB/DBManager.php(35): Helpers\DB\DBManager->__construct()
#2 /var/www/html/projet/app/Helpers/DB/Query.php(43): Helpers\DB\DBManager::getInstance()
#3 /var/www/html/projet/app/Controllers/Connexion.php(22): Helpers\DB\Query->__construct()
#4 /var/www/html/projet/app/Core/Router.php(124): Controllers\Connexion->__construct(NULL)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 04, 2016 20:51PM</p>

           <p><strong>Message:</strong> SQLSTATE[42S02]: Base table or view not found: 1146 Table 'projet.personne' doesn't exist SELECT * FROM personne WHERE login=?</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/Query.php(62): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Models/Queries/PersonneSQL.php(15): Helpers\DB\Query->execute()
#2 /var/www/html/projet/app/Controllers/Connexion.php(102): Models\Queries\PersonneSQL->prepareFindByLogin(NULL)
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 04, 2016 20:51PM</p>

           <p><strong>Message:</strong> SQLSTATE[42S02]: Base table or view not found: 1146 Table 'projet.personne' doesn't exist SELECT * FROM personne WHERE login=?</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/Query.php(62): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Models/Queries/PersonneSQL.php(15): Helpers\DB\Query->execute()
#2 /var/www/html/projet/app/Controllers/Connexion.php(102): Models\Queries\PersonneSQL->prepareFindByLogin(NULL)
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 04, 2016 20:51PM</p>

           <p><strong>Message:</strong> SQLSTATE[42S02]: Base table or view not found: 1146 Table 'projet.personne' doesn't exist SELECT * FROM personne WHERE login=?</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/Query.php(62): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Models/Queries/PersonneSQL.php(15): Helpers\DB\Query->execute()
#2 /var/www/html/projet/app/Controllers/Connexion.php(102): Models\Queries\PersonneSQL->prepareFindByLogin(NULL)
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 04, 2016 20:51PM</p>

           <p><strong>Message:</strong> SQLSTATE[42S02]: Base table or view not found: 1146 Table 'projet.personne' doesn't exist SELECT * FROM personne WHERE login=?</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/Query.php(62): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Models/Queries/PersonneSQL.php(15): Helpers\DB\Query->execute()
#2 /var/www/html/projet/app/Controllers/Connexion.php(102): Models\Queries\PersonneSQL->prepareFindByLogin(NULL)
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 04, 2016 20:52PM</p>

           <p><strong>Message:</strong> SQLSTATE[42S02]: Base table or view not found: 1146 Table 'projet.personne' doesn't exist SELECT * FROM personne WHERE login=?</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/Query.php(62): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Models/Queries/PersonneSQL.php(15): Helpers\DB\Query->execute()
#2 /var/www/html/projet/app/Controllers/Connexion.php(102): Models\Queries\PersonneSQL->prepareFindByLogin('Bonou')
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 04, 2016 20:54PM</p>

           <p><strong>Message:</strong> SQLSTATE[42S22]: Column not found: 1054 Unknown column 'login' in 'where clause' SELECT * FROM personne WHERE login=?</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/Query.php(62): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Models/Queries/PersonneSQL.php(15): Helpers\DB\Query->execute()
#2 /var/www/html/projet/app/Controllers/Connexion.php(102): Models\Queries\PersonneSQL->prepareFindByLogin('Bonou')
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 04, 2016 20:54PM</p>

           <p><strong>Message:</strong> SQLSTATE[42S22]: Column not found: 1054 Unknown column 'login' in 'where clause' SELECT * FROM personne WHERE login=?</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/Query.php(62): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Models/Queries/PersonneSQL.php(15): Helpers\DB\Query->execute()
#2 /var/www/html/projet/app/Controllers/Connexion.php(102): Models\Queries\PersonneSQL->prepareFindByLogin('Bonou')
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 04, 2016 20:54PM</p>

           <p><strong>Message:</strong> SQLSTATE[42S22]: Column not found: 1054 Unknown column 'login' in 'where clause' SELECT * FROM personne WHERE login=?</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/Query.php(62): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Models/Queries/PersonneSQL.php(15): Helpers\DB\Query->execute()
#2 /var/www/html/projet/app/Controllers/Connexion.php(102): Models\Queries\PersonneSQL->prepareFindByLogin('Bonou')
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 04, 2016 21:03PM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(71): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(127): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 04, 2016 21:04PM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(71): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(128): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 04, 2016 21:05PM</p>

           <p><strong>Message:</strong> SQLSTATE[42S22]: Column not found: 1054 Unknown column 'email' in 'where clause' SELECT * FROM personne WHERE email=?</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/Query.php(62): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Controllers/Connexion.php(116): Helpers\DB\Query->execute()
#2 [internal function]: Controllers\Connexion->inscription()
#3 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#4 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#5 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#6 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#7 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 04, 2016 21:06PM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(71): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(128): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 6:31AM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(71): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(128): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 6:33AM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(71): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(128): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 6:40AM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(72): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(128): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 6:43AM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(73): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(128): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 6:44AM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(72): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(128): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 6:44AM</p>

           <p><strong>Message:</strong> SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''')' at line 1 INSERT INTO personne VALUES(,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(72): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(128): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 6:44AM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(72): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(128): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 6:45AM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(73): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(128): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 6:46AM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(72): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(128): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 6:47AM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(71): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(128): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 6:48AM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(71): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(128): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 6:49AM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(71): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(128): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 6:52AM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?,?,?,?,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(72): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(128): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 6:53AM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?,?,?,?,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(73): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(128): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 6:54AM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(71): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(128): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 6:54AM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(71): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(129): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 7:01AM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(71): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(128): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 7:03AM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(71): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(128): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 7:03AM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(71): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(129): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 7:04AM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(71): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(129): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 7:06AM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(71): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(129): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 7:06AM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(71): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(129): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 7:09AM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(71): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(129): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 7:10AM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(72): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(129): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 7:11AM</p>

           <p><strong>Message:</strong> SQLSTATE[21S01]: Insert value list does not match column list: 1136 Column count doesn't match value count at row 1 INSERT INTO personne VALUES(NULL)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(72): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(129): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 7:15AM</p>

           <p><strong>Message:</strong> SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''TestInscription','$2y$10$KCjCySKk43p1TE1gTvctKO89WDOSjIZbXE76W9fEhKfSUwoXc.p.O'' at line 1 INSERT INTO personne VALUES(,?,?,?,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(72): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(131): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 7:16AM</p>

           <p><strong>Message:</strong> SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''TestInscription','$2y$10$g8Ml65kUGXtknYUuZmg5Z.uFauBbE5JLDAexr0YQ3VFXWFHfS7sZe'' at line 1 INSERT INTO personne VALUES(,?,?,?,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(73): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(131): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 7:17AM</p>

           <p><strong>Message:</strong> SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens INSERT INTO personne VALUES(?,?,?,?,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(73): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(131): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 7:17AM</p>

           <p><strong>Message:</strong> SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens INSERT INTO personne VALUES(?,?,?,?,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(73): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(131): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 7:18AM</p>

           <p><strong>Message:</strong> SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens INSERT INTO personne VALUES(?,?,?,?,?,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /var/www/html/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 /var/www/html/projet/app/Helpers/DB/EntityManager.php(73): Helpers\DB\DBManager->execute(Array)
#1 /var/www/html/projet/app/Helpers/DB/EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 /var/www/html/projet/app/Controllers/Connexion.php(131): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 /var/www/html/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#5 /var/www/html/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\Con...')
#6 /var/www/html/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /var/www/html/projet/index.php(81): require('/var/www/html/p...')
#8 {main}</pre>

           <hr />
<p>Error on Feb 05, 2016 7:51AM - Invalid argument supplied for foreach() in /var/www/html/projet/app/views/connexion/inscription.php on line 18</p><h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 17:50PM</p>

           <p><strong>Message:</strong> SQLSTATE[42S22]: Column not found: 1054 Champ 'pseudo' inconnu dans where clause SELECT * FROM personne WHERE pseudo=?</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> F:\wamp\www\projet\app\Helpers\DB\DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 F:\wamp\www\projet\app\Helpers\DB\Query.php(62): Helpers\DB\DBManager->execute(Array)
#1 F:\wamp\www\projet\app\Models\Queries\PersonneSQL.php(15): Helpers\DB\Query->execute()
#2 F:\wamp\www\projet\app\Controllers\Connexion.php(112): Models\Queries\PersonneSQL->prepareFindByLogin('Test')
#3 [internal function]: Controllers\Connexion->inscription()
#4 F:\wamp\www\projet\app\Core\Router.php(126): call_user_func_array(Array, Array)
#5 F:\wamp\www\projet\app\Core\Router.php(217): Core\Router::invokeObject('Controllers\\Con...')
#6 F:\wamp\www\projet\app\Core\routes.php(35): Core\Router::dispatch()
#7 F:\wamp\www\projet\index.php(81): require('F:\\wamp\\www\\pro...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 05, 2016 17:55PM</p>

           <p><strong>Message:</strong> SQLSTATE[42S22]: Column not found: 1054 Champ 'email' inconnu dans where clause SELECT * FROM personne WHERE email=?</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> F:\wamp\www\projet\app\Helpers\DB\DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 F:\wamp\www\projet\app\Helpers\DB\Query.php(62): Helpers\DB\DBManager->execute(Array)
#1 F:\wamp\www\projet\app\Controllers\Connexion.php(120): Helpers\DB\Query->execute()
#2 [internal function]: Controllers\Connexion->inscription()
#3 F:\wamp\www\projet\app\Core\Router.php(126): call_user_func_array(Array, Array)
#4 F:\wamp\www\projet\app\Core\Router.php(217): Core\Router::invokeObject('Controllers\\Con...')
#5 F:\wamp\www\projet\app\Core\routes.php(35): Core\Router::dispatch()
#6 F:\wamp\www\projet\index.php(81): require('F:\\wamp\\www\\pro...')
#7 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 25, 2016 15:31PM</p>

           <p><strong>Message:</strong> SQLSTATE[HY000] [1045] Access denied for user 'toto'@'localhost' (using password: YES)</p>

           <p><strong>Code:</strong> 1045</p>

           <p><strong>File:</strong> /Applications/MAMP/htdocs/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 22</p>

           <h3>Stack trace:</h3>

           <pre>#0 /Applications/MAMP/htdocs/projet/app/Helpers/DB/DBManager.php(22): PDO->__construct('mysql:host=loca...', 'toto', '********', Array)
#1 /Applications/MAMP/htdocs/projet/app/Helpers/DB/DBManager.php(35): Helpers\DB\DBManager->__construct()
#2 /Applications/MAMP/htdocs/projet/app/Helpers/DB/Query.php(43): Helpers\DB\DBManager::getInstance()
#3 /Applications/MAMP/htdocs/projet/app/Controllers/Connexion.php(29): Helpers\DB\Query->__construct()
#4 /Applications/MAMP/htdocs/projet/app/Core/Router.php(124): Controllers\Connexion->__construct(NULL)
#5 /Applications/MAMP/htdocs/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\\Con...')
#6 /Applications/MAMP/htdocs/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /Applications/MAMP/htdocs/projet/index.php(81): require('/Applications/M...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Feb 29, 2016 21:39PM</p>

           <p><strong>Message:</strong> SQLSTATE[42S22]: Column not found: 1054 Champ 'mail' inconnu dans field list UPDATE personne SET pseudo = ?,motdepasse = ?,mail = ?,cookie = ?,id = ?WHERE id=?</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> F:\wamp\www\projet\app\Helpers\DB\DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 F:\wamp\www\projet\app\Helpers\DB\EntityManager.php(99): Helpers\DB\DBManager->execute(Array)
#1 F:\wamp\www\projet\app\Helpers\DB\EntityManager.php(45): Helpers\DB\EntityManager->update(Object(Models\Tables\Personne))
#2 F:\wamp\www\projet\app\Controllers\Connexion.php(66): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->connexion()
#4 F:\wamp\www\projet\app\Core\Router.php(126): call_user_func_array(Array, Array)
#5 F:\wamp\www\projet\app\Core\Router.php(217): Core\Router::invokeObject('Controllers\\Con...')
#6 F:\wamp\www\projet\app\Core\routes.php(35): Core\Router::dispatch()
#7 F:\wamp\www\projet\index.php(81): require('F:\\wamp\\www\\pro...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Mar 02, 2016 23:57PM</p>

           <p><strong>Message:</strong> SQLSTATE[HY000] [1045] Access denied for user 'toto'@'localhost' (using password: YES)</p>

           <p><strong>Code:</strong> 1045</p>

           <p><strong>File:</strong> /Applications/MAMP/htdocs/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 22</p>

           <h3>Stack trace:</h3>

           <pre>#0 /Applications/MAMP/htdocs/projet/app/Helpers/DB/DBManager.php(22): PDO->__construct('mysql:host=loca...', 'toto', '********', Array)
#1 /Applications/MAMP/htdocs/projet/app/Helpers/DB/DBManager.php(35): Helpers\DB\DBManager->__construct()
#2 /Applications/MAMP/htdocs/projet/app/Helpers/DB/Query.php(43): Helpers\DB\DBManager::getInstance()
#3 /Applications/MAMP/htdocs/projet/app/Controllers/Connexion.php(29): Helpers\DB\Query->__construct()
#4 /Applications/MAMP/htdocs/projet/app/Core/Router.php(124): Controllers\Connexion->__construct(NULL)
#5 /Applications/MAMP/htdocs/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\\Con...')
#6 /Applications/MAMP/htdocs/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /Applications/MAMP/htdocs/projet/index.php(81): require('/Applications/M...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Mar 03, 2016 7:21AM</p>

           <p><strong>Message:</strong> SQLSTATE[HY000] [1045] Access denied for user 'toto'@'localhost' (using password: YES)</p>

           <p><strong>Code:</strong> 1045</p>

           <p><strong>File:</strong> /Applications/MAMP/htdocs/projet/app/Helpers/DB/DBManager.php</p>

           <p><strong>Line:</strong> 22</p>

           <h3>Stack trace:</h3>

           <pre>#0 /Applications/MAMP/htdocs/projet/app/Helpers/DB/DBManager.php(22): PDO->__construct('mysql:host=loca...', 'toto', '********', Array)
#1 /Applications/MAMP/htdocs/projet/app/Helpers/DB/DBManager.php(35): Helpers\DB\DBManager->__construct()
#2 /Applications/MAMP/htdocs/projet/app/Helpers/DB/Query.php(43): Helpers\DB\DBManager::getInstance()
#3 /Applications/MAMP/htdocs/projet/app/Controllers/Connexion.php(29): Helpers\DB\Query->__construct()
#4 /Applications/MAMP/htdocs/projet/app/Core/Router.php(124): Controllers\Connexion->__construct(NULL)
#5 /Applications/MAMP/htdocs/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\\Con...')
#6 /Applications/MAMP/htdocs/projet/app/Core/routes.php(35): Core\Router::dispatch()
#7 /Applications/MAMP/htdocs/projet/index.php(81): require('/Applications/M...')
#8 {main}</pre>

           <hr />
<p>Error on Mar 04, 2016 21:48PM - Division by zero in C:\wamp\www\projet\app\templates\default\header.php on line 51</p><p>Error on Mar 04, 2016 21:48PM - Division by zero in C:\wamp\www\projet\app\templates\default\header.php on line 51</p><p>Error on Mar 04, 2016 21:50PM - Division by zero in C:\wamp\www\projet\app\templates\default\header.php on line 51</p><p>Error on Mar 04, 2016 21:50PM - Division by zero in C:\wamp\www\projet\app\templates\default\header.php on line 51</p><p>Error on Mar 04, 2016 21:50PM - Division by zero in C:\wamp\www\projet\app\templates\default\header.php on line 50</p><h3>Exception information:</h3>

           <p><strong>Date:</strong> Mar 12, 2016 14:23PM</p>

           <p><strong>Message:</strong> syntax error, unexpected end of file</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /Applications/MAMP/htdocs/projet/app/views/regles/regles.php</p>

           <p><strong>Line:</strong> 178</p>

           <h3>Stack trace:</h3>

           <pre>#0 /Applications/MAMP/htdocs/projet/app/Controllers/Regles.php(33): Core\View::render('regles/regles', Array)
#1 [internal function]: Controllers\Regles->index()
#2 /Applications/MAMP/htdocs/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#3 /Applications/MAMP/htdocs/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\\Reg...')
#4 /Applications/MAMP/htdocs/projet/app/Core/routes.php(37): Core\Router::dispatch()
#5 /Applications/MAMP/htdocs/projet/index.php(81): require('/Applications/M...')
#6 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Mar 12, 2016 14:23PM</p>

           <p><strong>Message:</strong> syntax error, unexpected end of file</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /Applications/MAMP/htdocs/projet/app/views/regles/regles.php</p>

           <p><strong>Line:</strong> 178</p>

           <h3>Stack trace:</h3>

           <pre>#0 /Applications/MAMP/htdocs/projet/app/Controllers/Regles.php(33): Core\View::render('regles/regles', Array)
#1 [internal function]: Controllers\Regles->index()
#2 /Applications/MAMP/htdocs/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#3 /Applications/MAMP/htdocs/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\\Reg...')
#4 /Applications/MAMP/htdocs/projet/app/Core/routes.php(37): Core\Router::dispatch()
#5 /Applications/MAMP/htdocs/projet/index.php(81): require('/Applications/M...')
#6 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Mar 12, 2016 14:23PM</p>

           <p><strong>Message:</strong> syntax error, unexpected end of file</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /Applications/MAMP/htdocs/projet/app/views/regles/regles.php</p>

           <p><strong>Line:</strong> 178</p>

           <h3>Stack trace:</h3>

           <pre>#0 /Applications/MAMP/htdocs/projet/app/Controllers/Regles.php(33): Core\View::render('regles/regles', Array)
#1 [internal function]: Controllers\Regles->index()
#2 /Applications/MAMP/htdocs/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#3 /Applications/MAMP/htdocs/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\\Reg...')
#4 /Applications/MAMP/htdocs/projet/app/Core/routes.php(37): Core\Router::dispatch()
#5 /Applications/MAMP/htdocs/projet/index.php(81): require('/Applications/M...')
#6 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Mar 12, 2016 14:25PM</p>

           <p><strong>Message:</strong> syntax error, unexpected end of file</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /Applications/MAMP/htdocs/projet/app/views/regles/regles.php</p>

           <p><strong>Line:</strong> 178</p>

           <h3>Stack trace:</h3>

           <pre>#0 /Applications/MAMP/htdocs/projet/app/Controllers/Regles.php(33): Core\View::render('regles/regles', Array)
#1 [internal function]: Controllers\Regles->index()
#2 /Applications/MAMP/htdocs/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#3 /Applications/MAMP/htdocs/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\\Reg...')
#4 /Applications/MAMP/htdocs/projet/app/Core/routes.php(37): Core\Router::dispatch()
#5 /Applications/MAMP/htdocs/projet/index.php(81): require('/Applications/M...')
#6 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Mar 12, 2016 14:25PM</p>

           <p><strong>Message:</strong> syntax error, unexpected end of file</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /Applications/MAMP/htdocs/projet/app/views/regles/regles.php</p>

           <p><strong>Line:</strong> 178</p>

           <h3>Stack trace:</h3>

           <pre>#0 /Applications/MAMP/htdocs/projet/app/Controllers/Regles.php(33): Core\View::render('regles/regles', Array)
#1 [internal function]: Controllers\Regles->index()
#2 /Applications/MAMP/htdocs/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#3 /Applications/MAMP/htdocs/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\\Reg...')
#4 /Applications/MAMP/htdocs/projet/app/Core/routes.php(37): Core\Router::dispatch()
#5 /Applications/MAMP/htdocs/projet/index.php(81): require('/Applications/M...')
#6 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Mar 12, 2016 16:34PM</p>

           <p><strong>Message:</strong> Class 'Session' not found</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /Applications/MAMP/htdocs/projet/app/views/play/playIndex.php</p>

           <p><strong>Line:</strong> 5</p>

           <h3>Stack trace:</h3>

           <pre>#0 /Applications/MAMP/htdocs/projet/app/Core/View.php(34): require()
#1 /Applications/MAMP/htdocs/projet/app/Controllers/Play.php(25): Core\View::render('play/playIndex', Array)
#2 [internal function]: Controllers\Play->index()
#3 /Applications/MAMP/htdocs/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#4 /Applications/MAMP/htdocs/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\\Pla...')
#5 /Applications/MAMP/htdocs/projet/app/Core/routes.php(37): Core\Router::dispatch()
#6 /Applications/MAMP/htdocs/projet/index.php(81): require('/Applications/M...')
#7 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Mar 12, 2016 16:35PM</p>

           <p><strong>Message:</strong> Class 'Session' not found</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /Applications/MAMP/htdocs/projet/app/views/play/playIndex.php</p>

           <p><strong>Line:</strong> 7</p>

           <h3>Stack trace:</h3>

           <pre>#0 /Applications/MAMP/htdocs/projet/app/Core/View.php(34): require()
#1 /Applications/MAMP/htdocs/projet/app/Controllers/Play.php(25): Core\View::render('play/playIndex', Array)
#2 [internal function]: Controllers\Play->index()
#3 /Applications/MAMP/htdocs/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#4 /Applications/MAMP/htdocs/projet/app/Core/Router.php(217): Core\Router::invokeObject('Controllers\\Pla...')
#5 /Applications/MAMP/htdocs/projet/app/Core/routes.php(37): Core\Router::dispatch()
#6 /Applications/MAMP/htdocs/projet/index.php(81): require('/Applications/M...')
#7 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Mar 12, 2016 17:54PM</p>

           <p><strong>Message:</strong> syntax error, unexpected '""' (T_CONSTANT_ENCAPSED_STRING)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /Applications/MAMP/htdocs/projet/app/Controllers/Play.php</p>

           <p><strong>Line:</strong> 29</p>

           <h3>Stack trace:</h3>

           <pre>#0 /Applications/MAMP/htdocs/projet/vendor/composer/ClassLoader.php(301): Composer\Autoload\includeFile('/Applications/M...')
#1 [internal function]: Composer\Autoload\ClassLoader->loadClass('Controllers\\Pla...')
#2 /Applications/MAMP/htdocs/projet/app/Core/Router.php(124): spl_autoload_call('Controllers\\Pla...')
#3 /Applications/MAMP/htdocs/projet/app/Core/Router.php(254): Core\Router::invokeObject('Controllers\\Pla...', Array)
#4 /Applications/MAMP/htdocs/projet/app/Core/routes.php(38): Core\Router::dispatch()
#5 /Applications/MAMP/htdocs/projet/index.php(81): require('/Applications/M...')
#6 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Mar 12, 2016 18:07PM</p>

           <p><strong>Message:</strong> Class 'Session' not found</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /Applications/MAMP/htdocs/projet/app/views/play/playLevel.php</p>

           <p><strong>Line:</strong> 7</p>

           <h3>Stack trace:</h3>

           <pre>#0 /Applications/MAMP/htdocs/projet/app/Core/View.php(34): require()
#1 /Applications/MAMP/htdocs/projet/app/Controllers/Play.php(35): Core\View::render('play/playLevel', Array)
#2 [internal function]: Controllers\Play->playLevel('1')
#3 /Applications/MAMP/htdocs/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#4 /Applications/MAMP/htdocs/projet/app/Core/Router.php(254): Core\Router::invokeObject('Controllers\\Pla...', Array)
#5 /Applications/MAMP/htdocs/projet/app/Core/routes.php(38): Core\Router::dispatch()
#6 /Applications/MAMP/htdocs/projet/index.php(81): require('/Applications/M...')
#7 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Mar 12, 2016 18:07PM</p>

           <p><strong>Message:</strong> Class 'Session' not found</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> /Applications/MAMP/htdocs/projet/app/views/play/playLevel.php</p>

           <p><strong>Line:</strong> 7</p>

           <h3>Stack trace:</h3>

           <pre>#0 /Applications/MAMP/htdocs/projet/app/Core/View.php(34): require()
#1 /Applications/MAMP/htdocs/projet/app/Controllers/Play.php(35): Core\View::render('play/playLevel', Array)
#2 [internal function]: Controllers\Play->playLevel('1')
#3 /Applications/MAMP/htdocs/projet/app/Core/Router.php(126): call_user_func_array(Array, Array)
#4 /Applications/MAMP/htdocs/projet/app/Core/Router.php(254): Core\Router::invokeObject('Controllers\\Pla...', Array)
#5 /Applications/MAMP/htdocs/projet/app/Core/routes.php(38): Core\Router::dispatch()
#6 /Applications/MAMP/htdocs/projet/index.php(81): require('/Applications/M...')
#7 {main}</pre>

           <hr />
<p>Error on Mar 13, 2016 14:20PM - Invalid argument supplied for foreach() in F:\wamp\www\projet\app\views\scores\scores.php on line 17</p><p>Error on Mar 13, 2016 14:20PM - Invalid argument supplied for foreach() in F:\wamp\www\projet\app\views\scores\scores.php on line 17</p><p>Error on Mar 13, 2016 14:20PM - Invalid argument supplied for foreach() in F:\wamp\www\projet\app\views\scores\scores.php on line 17</p><p>Error on Mar 13, 2016 14:20PM - Invalid argument supplied for foreach() in F:\wamp\www\projet\app\views\scores\scores.php on line 17</p><p>Error on Mar 13, 2016 14:20PM - Invalid argument supplied for foreach() in F:\wamp\www\projet\app\views\scores\scores.php on line 17</p><p>Error on Mar 13, 2016 14:20PM - Invalid argument supplied for foreach() in F:\wamp\www\projet\app\views\scores\scores.php on line 17</p><h3>Exception information:</h3>

           <p><strong>Date:</strong> Mar 13, 2016 14:35PM</p>

           <p><strong>Message:</strong> SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicata du champ '1' pour la clef 'id' INSERT INTO personne VALUES(?,?,?,?,?,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> F:\wamp\www\projet\app\Helpers\DB\DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 F:\wamp\www\projet\app\Helpers\DB\EntityManager.php(73): Helpers\DB\DBManager->execute(Array)
#1 F:\wamp\www\projet\app\Helpers\DB\EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 F:\wamp\www\projet\app\Controllers\Connexion.php(136): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 F:\wamp\www\projet\app\Core\Router.php(126): call_user_func_array(Array, Array)
#5 F:\wamp\www\projet\app\Core\Router.php(217): Core\Router::invokeObject('Controllers\\Con...')
#6 F:\wamp\www\projet\app\Core\routes.php(38): Core\Router::dispatch()
#7 F:\wamp\www\projet\index.php(81): require('F:\\wamp\\www\\pro...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Mar 13, 2016 14:42PM</p>

           <p><strong>Message:</strong> SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicata du champ '1' pour la clef 'id' INSERT INTO personne VALUES(?,?,?,?,?,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> F:\wamp\www\projet\app\Helpers\DB\DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 F:\wamp\www\projet\app\Helpers\DB\EntityManager.php(73): Helpers\DB\DBManager->execute(Array)
#1 F:\wamp\www\projet\app\Helpers\DB\EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 F:\wamp\www\projet\app\Controllers\Connexion.php(136): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 F:\wamp\www\projet\app\Core\Router.php(126): call_user_func_array(Array, Array)
#5 F:\wamp\www\projet\app\Core\Router.php(217): Core\Router::invokeObject('Controllers\\Con...')
#6 F:\wamp\www\projet\app\Core\routes.php(38): Core\Router::dispatch()
#7 F:\wamp\www\projet\index.php(81): require('F:\\wamp\\www\\pro...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Mar 13, 2016 14:47PM</p>

           <p><strong>Message:</strong> SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicata du champ '1' pour la clef 'id' INSERT INTO personne VALUES(?,?,?,?,?,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> F:\wamp\www\projet\app\Helpers\DB\DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 F:\wamp\www\projet\app\Helpers\DB\EntityManager.php(73): Helpers\DB\DBManager->execute(Array)
#1 F:\wamp\www\projet\app\Helpers\DB\EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 F:\wamp\www\projet\app\Controllers\Connexion.php(136): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 F:\wamp\www\projet\app\Core\Router.php(126): call_user_func_array(Array, Array)
#5 F:\wamp\www\projet\app\Core\Router.php(217): Core\Router::invokeObject('Controllers\\Con...')
#6 F:\wamp\www\projet\app\Core\routes.php(38): Core\Router::dispatch()
#7 F:\wamp\www\projet\index.php(81): require('F:\\wamp\\www\\pro...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Mar 13, 2016 14:57PM</p>

           <p><strong>Message:</strong> SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicata du champ '1' pour la clef 'id' INSERT INTO personne VALUES(?,?,?,?,?,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> F:\wamp\www\projet\app\Helpers\DB\DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 F:\wamp\www\projet\app\Helpers\DB\EntityManager.php(73): Helpers\DB\DBManager->execute(Array)
#1 F:\wamp\www\projet\app\Helpers\DB\EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 F:\wamp\www\projet\app\Controllers\Connexion.php(136): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 F:\wamp\www\projet\app\Core\Router.php(126): call_user_func_array(Array, Array)
#5 F:\wamp\www\projet\app\Core\Router.php(217): Core\Router::invokeObject('Controllers\\Con...')
#6 F:\wamp\www\projet\app\Core\routes.php(38): Core\Router::dispatch()
#7 F:\wamp\www\projet\index.php(81): require('F:\\wamp\\www\\pro...')
#8 {main}</pre>

           <hr />
<h3>Exception information:</h3>

           <p><strong>Date:</strong> Mar 14, 2016 9:56AM</p>

           <p><strong>Message:</strong> SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicata du champ '1' pour la clef 'id' INSERT INTO personne VALUES(?,?,?,?,?,?)</p>

           <p><strong>Code:</strong> 0</p>

           <p><strong>File:</strong> F:\wamp\www\projet\app\Helpers\DB\DBManager.php</p>

           <p><strong>Line:</strong> 59</p>

           <h3>Stack trace:</h3>

           <pre>#0 F:\wamp\www\projet\app\Helpers\DB\EntityManager.php(73): Helpers\DB\DBManager->execute(Array)
#1 F:\wamp\www\projet\app\Helpers\DB\EntityManager.php(43): Helpers\DB\EntityManager->insert(Object(Models\Tables\Personne))
#2 F:\wamp\www\projet\app\Controllers\Connexion.php(136): Helpers\DB\EntityManager->save(Object(Models\Tables\Personne))
#3 [internal function]: Controllers\Connexion->inscription()
#4 F:\wamp\www\projet\app\Core\Router.php(126): call_user_func_array(Array, Array)
#5 F:\wamp\www\projet\app\Core\Router.php(217): Core\Router::invokeObject('Controllers\\Con...')
#6 F:\wamp\www\projet\app\Core\routes.php(38): Core\Router::dispatch()
#7 F:\wamp\www\projet\index.php(81): require('F:\\wamp\\www\\pro...')
#8 {main}</pre>

           <hr />
